src/components/Modal/confirmationModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+import './style.scss';
+
+export default class ConfirmationModal extends React.Component {
+  render() {
+    const {
+      title, message, submit, cancel, style,
+    } = this.props;
+    return (
+      <div className="modal-wrapper">
+        <div className="confirmation-modal-container" style={style || null}>
+          <div style={{ padding: '30px' }}>
```

Comment:
Same as @alexanderqchen – either give them classes or use css selectors to style these